sklearn/linear_model/ridge.py

```diff
@@ -268,6 +263,11 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
         .. versionadded:: 0.19
            SAGA solver.
 
+    max_iter : int, optional
+        Maximum number of iterations for conjugate gradient solver.
+        For 'sparse_cg' and 'lsqr' solvers, the default value is determined
+        by scipy.sparse.linalg. For 'sag' solver, the default value is 1000.
```

Comment:
I assume this is also the case for the 'saga' solver:

For the "sag" and "saga" solvers, ...